sphinx_gallery/components/sg_launcher_links.html

```diff
@@ -0,0 +1,11 @@
+{# Displays the launcher badges for the example. #}
+{% set launcher_links = get_launcher_links() %}
+{% for key, link_info in launcher_links.items() %}
+  {% if link_info %}
+    <div class="{{ key }}-badge-sidebar">
+      <a href="{{ link_info.link }}">
+        <img src="{{ link_info.img_src }}" alt="{{ link_info.img_alt }}" height="20">
```

Comment:
Rather than have a static `height` here it would probably be better to use a CSS class on the image or a parent (like the `div`), and add an entry in [SG's CSS](https://github.com/sphinx-gallery/sphinx-gallery/blob/master/sphinx_gallery/_static/sg_gallery.css) to set the height. That way end-users can override the value if they want using CSS.